Makefile.dune

```diff
@@ -10,10 +10,11 @@ BUILD_CONTEXT=_build/default
 
 help:
 	@echo "Welcome to Coq's Dune-based build system. Targets are:"
-	@echo "  - states: build a minimal functional coqtop"
-	@echo "  - world:  build all binaries and libraries"
-	@echo "  - clean:  remove build directory and autogenerated files"
-	@echo "  - help:   show this message"
+	@echo "  - states:  build a minimal functional coqtop"
+	@echo "  - world:   build all binaries and libraries"
+	@echo "  - release: build Coq in release mode"
+	@echo "  - clean:   remove build directory and autogenerated files"
+	@echo "  - help:    show this message"
```

Comment:
Do we really want to align stuff here? It ends up making updates annoying.

Reply:
Given that this is a document I consider formatting important. The alternative would be to use a table generator.